pom.xml

```diff
@@ -404,6 +400,27 @@
                     </configuration>
                 </plugin>
                 
+                <!-- Configure checkstyle with sensible defaults for this project so it can be
+                     executed standalone from the CLI as follows:
+                     
+                         mvn checkstyle:check 
+                 -->
+	            <plugin>
+	                <groupId>org.apache.maven.plugins</groupId>
+	                <artifactId>maven-checkstyle-plugin</artifactId>
+	                <version>${maven-checkstyle-plugin.version}</version>
+	                <configuration>
+	                    <configLocation>src/checkstyle/checkstyle.xml</configLocation>
+	                </configuration>
+	                <dependencies>
+						<dependency>
+							<groupId>com.puppycrawl.tools</groupId>
+	              			<artifactId>checkstyle</artifactId>
+	              			<version>${checkstyle.version}</version>
+	            		</dependency>
+	                </dependencies>
+	            </plugin>
+	            
```

Comment:
replace tabs with spaces to keep consistent with the rest of the file.

(Kind of ironic since this is a checkstyle review.  hah!)